src/Akeneo/Pim/Structure/Component/Security/AssociationTypeScopeMapper.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Pim\Structure\Component\Security;
+
+use Akeneo\Connectivity\Connection\Infrastructure\Apps\Security\ScopeMapperInterface;
+
+/**
+ * @copyright 2021 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class AssociationTypeScopeMapper implements ScopeMapperInterface
```

Comment:
```suggestion
class AssociationTypesScopeMapper implements ScopeMapperInterface
```
with or without `s`, I don't know what is correct... scope names have an `s`, acls don't...

Reply:
good question : the model and the interface are without `s`. I can change it if needed.